migration-tool/src/test/java/software/amazon/awssdk/migration/recipe/ChangeSdkTypeTest.java

```diff
@@ -215,4 +216,30 @@ void shouldChangeFieldsInInnerClass() {
             )
         );
     }
+
+    @Test
+    @EnabledOnJre({JRE.JAVA_8})
+    void hasUnsupportedFeature_shouldSkip() {
```

Comment:
nit : TODO to update/replace test

Reply:
Test will fail right away once we add support :)